### PR TITLE
Fix scoping issues for client-mu-plugins

### LINKED ISSUE #668: client-mu-plugins: Fix scoping issues
Because we load the plugins inside a function, we're not giving `client-mu-plugins` the same global scope that `mu-plugins` get. 

We could simplify this (and the entire loading process) by getting rid of the function and just doing the loading the files directly inside the mu-plugins script.

props @fklein-lu ([#](https://github.com/Automattic/vip-go-mu-plugins/pull/611#issuecomment-311002566))

### PR DESCRIPTION
**Pull request by mjangda**
Don't load them inside a function. Instead, load them in global scope inside the mu-plugin loader so the the loaded client-mu-plugins also have global scope.

Fixes #668

### CODE REVIEW

File: z-client-mu-plugins.php [hunk 1]
```diff
@@ -1,107 +1,85 @@
...
+	if ( empty( $client_mu_plugins_files ) ) {
+		return [];
 	}
 
-	foreach ( $plugin_files as $plugin_file ) {
-		if ( ! is_readable( WPCOM_VIP_CLIENT_MU_PLUGIN_DIR . "/$plugin_file" ) ) {
+	$client_mu_plugins_data = [];
+
+	foreach ( $client_mu_plugins_files as $plugin_file ) {
+		if ( ! is_readable( WPCOM_VIP_CLIENT_MU_PLUGIN_DIR . '/' . $plugin_file ) ) {
```
Comment (scottsweb) [inline z-client-mu-plugins.php]: For some reason this path is a little strange for me when running this locally: 

`/home/scott/Sites/wordpress/wp-content/client-mu-plugins//home/scott/Sites/wordpress/wp-content/client-mu-plugins/test-plugin.php`

Not sure what has really changed (could be something on my local setup) but it is causing the endpoint to fail for me.

Comment (mjangda) [inline z-client-mu-plugins.php]: Hm, that looks broken. Will take a look.

Comment (mjangda) [inline z-client-mu-plugins.php]: Good catch; the return signatures were a little different between the function I have and the one we were originally using. Should be fixed with defec1d.

Comment (mjangda) [inline z-client-mu-plugins.php]: Hm, probably should add tests...

**Comment by mjangda** [comment]
r95962-deploy

**Pull request merged**